Client/core/CCore.cpp

```diff
@@ -1957,6 +1968,24 @@ uint CCore::GetMaxStreamingMemory()
     return m_fMaxStreamingMemory;
 }
 
+//
+// ResetDiscordRichPresence
+//
+void CCore::ResetDiscordRichPresence()
+{
+    if (m_pDiscordManager)
```

Comment:
The else branch can't happen (unless you delete `CCore` and then call this funciton, which is going to blow up anyway).